app/MMB/src/scenes/tripServices/InsuranceMenuItem.js

```diff
@@ -24,8 +77,43 @@ export default class InsuranceMenuItem extends React.Component<Props> {
           <Translation id="mmb.trip_services.general_services.insurance" />
         }
         icon={<TextIcon code="'" />}
-        onPress={this.props.onOpenInsurance}
+        onPress={this.onPress}
       />
     );
   };
 }
+
+export default createFragmentContainer(
+  InsuranceMenuItem,
+  graphql`
+    fragment InsuranceMenuItem on BookingInterface {
+      __typename
+      status
+      passengers {
+        nationality
+      }
+      isPastBooking
+      ... on BookingOneWay {
+        trip {
+          departure {
+            localTime
+          }
+        }
+      }
+      ... on BookingReturn {
+        outbound {
+          departure {
+            localTime
+          }
+        }
+      }
+      ... on BookingMulticity {
+        trips {
+          departure {
+            localTime
+          }
```

Comment:
this pattern is repeating
```
departure {
  localTime
}
```

it could be extracted to a new fragment and be passed date from this component accordingly.